routes/htmlRoutes.js

```diff
@@ -40,15 +40,33 @@ router.get("/login", (req, res) => {
 // Here we've add our isAuthenticated middleware to this route.
 // If a user who is not logged in tries to access this route they will be redirected to the signup page
 router.get("/members", isAuthenticated, (req, res) => {
-  if (
-    req.user.isDeveloper &&
-    !req.session.hasAlreadyBeenRedirectedFromMembersToDevelopersOnce
-  ) {
-    req.session.hasAlreadyBeenRedirectedFromMembersToDevelopersOnce = true;
-    res.redirect("/developers");
-  } else {
-    res.sendFile(path.join(__dirname, "../public/members.html"));
-  }
+  const messageUsername = req.user.username;
+  const usernameToLookAt = req.user.username; 
+
```

Comment:
This logic seems all good and find at a glance but for the sake of organization, I would suggest that meat of the actual logic that asks the model for the required info should be put into some separate api file to keep the routing logic focused on JUST routing.